ocs_ci/ocs/ocp.py

```diff
@@ -555,6 +555,13 @@ def wait_for_resource(
                 f"Wait for {self._kind} resource {resource_name} at column {column}"
                 f" to reach desired condition {condition} failed,"
                 f" last actual status was {actual_status}"))
+            # run `oc describe` on the resources we were waiting for to provide
```

Comment:
Nice!

I thought we already have it but I double-checked and this exists only for wait_for_delete() method